### PR TITLE
Fix issue #469 - make polyfill to be internal

### LINKED ISSUE #469: Ensure any third-party scripts that are used are hosted locally and not dynamically referenced from a third party site
Action from #82 

GA and Optimizely are exceptions to this, as we have contracts in place to manage the protection of data shared with those providers.

### PR DESCRIPTION
**Pull request by alicoding**
closes #469

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

File: pages/index.jsx [hunk 1]
```diff
@@ -27,12 +27,12 @@ var Index = React.createClass({
...
           <div id="my-app" dangerouslySetInnerHTML={{__html: this.props.markup}}></div>
-          <script src="https://cdn.polyfill.io/v1/polyfill.min.js?features=Promise,Intl.~locale.fr,Intl.~locale.pt-BR,Intl.~locale.de,Intl.~locale.id,Intl.~locale.es,Intl.~locale.en-US"></script>
+          <script src="/api/polyfill"></script>
           <script src="/main.js"></script>
           <Pontoon/>
           {/*[if lt IE 10]*/}
-          <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js"></script>
-          <script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/4.0.1/placeholders.jquery.js"></script>
+          <script src="/js/jquery-1.9.1.js"></script>
+          <script src="/js/placeholders.jquery.js"></script>
```
Comment (ScottDowne) [inline pages/index.jsx]: No npm for this?

File: pages/index.jsx [hunk 2]
```diff
@@ -27,12 +27,12 @@ var Index = React.createClass({
...
         <body>
           <div id="my-app" dangerouslySetInnerHTML={{__html: this.props.markup}}></div>
-          <script src="https://cdn.polyfill.io/v1/polyfill.min.js?features=Promise,Intl.~locale.fr,Intl.~locale.pt-BR,Intl.~locale.de,Intl.~locale.id,Intl.~locale.es,Intl.~locale.en-US"></script>
+          <script src="/api/polyfill"></script>
           <script src="/main.js"></script>
           <Pontoon/>
           {/*[if lt IE 10]*/}
-          <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.9.1/jquery.js"></script>
-          <script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/4.0.1/placeholders.jquery.js"></script>
+          <script src="/js/jquery-1.9.1.js"></script>
```
Comment (ScottDowne) [inline pages/index.jsx]: No npm for this?

**Comment by ScottDowne** [comment]
This is fine. My comment isn't critical.

**Pull request merged**